### PR TITLE
[core] Minor refactoring during the code studying

### PR DESCRIPTION
**Pull request by mbakholdina**
Minor refactoring during the code studying.

### CODE REVIEW

File: srtcore/core.h [hunk 1]
```diff
@@ -97,22 +97,23 @@ enum AckDataItem
...
-    ACKD_TOTAL_SIZE_VER101 = 7, // length = 28
-    ACKD_XMRATE = 7, // XXX This is a weird compat stuff. Version 1.1.3 defines it as ACKD_BANDWIDTH*m_iMaxSRTPayloadSize when set. Never got.
-                     // XXX NOTE: field number 7 may be used for something in future, need to confirm destruction of all !compat 1.0.2 version
-
-    ACKD_TOTAL_SIZE_VER102 = 8, // 32
-// FEATURE BLOCKED. Probably not to be restored.
-//  ACKD_ACKBITMAP = 8,
-    ACKD_TOTAL_SIZE = ACKD_TOTAL_SIZE_VER102 // length = 32 (or more)
+    ACKD_TOTAL_SIZE_VER101 = 7,  // length = 28
+    // XXX This is a weird compat stuff. Version 1.1.3 defines it as
```
Comment (mbakholdina) [inline srtcore/core.h:108]: @maxsharabayko Please provide me better explanation for this TODO section, I'll update appropriately.

File: srtcore/core.h [hunk 2]
```diff
@@ -97,22 +97,23 @@ enum AckDataItem
...
+    // ACKD_BANDWIDTH*m_iMaxSRTPayloadSize when set. Never got.
+    // XXX NOTE: field number 7 may be used for something in future,
+    // need to confirm destruction of all !compat 1.0.2 version.
+    ACKD_XMRATE = 7,
```
Comment (maxsharabayko) [inline srtcore/core.h:112]: I would maybe rename `ACKD_XMRATE` into `ACKD_VER102_XMRATE`.
Place it together with `ACKD_TOTAL_SIZE_VER102`. 

And remove those comments.

Should look like so:

```c++
enum AckDataItem
{
    ACKD_RCVLASTACK       = 0,
    ACKD_RTT              = 1,
    ACKD_RTTVAR           = 2,
    ACKD_BUFFERLEFT       = 3,
    ACKD_TOTAL_SIZE_SMALL = 4, // Size of the Small ACK, packet length = 16.

    // Extra fields for Full ACK
    ACKD_RCVSPEED           = 4,
    ACKD_BANDWIDTH          = 5,
    ACKD_TOTAL_SIZE_UDTBASE = 6, // Packet length = 24.

    // Extra stats after SRT v1.0.1
    ACKD_RCVRATE           = 6,
    ACKD_TOTAL_SIZE_VER101 = 7, // Packet length = 28.

    // Only in SRT v 1.0.2.
    ACKD_VER102_ONLY_XMRATE     = 7,
    ACKD_TOTAL_SIZE_VER102_ONLY = 8, // Packet length = 32.

    ACKD_TOTAL_SIZE = ACKD_TOTAL_SIZE_VER102 // The maximum known ACK length is 32 bytes.
};
```

File: srtcore/core.cpp [hunk 3]
```diff
@@ -8001,10 +8005,10 @@ void CUDT::processCtrlAck(const CPacket &ctrlpkt, const steady_clock::time_point
      * Additional UDT fields, not always attached:
      *   ACKD_RCVSPEED
      *   ACKD_BANDWIDTH
-     * SRT extension version 1.0.2 (bstats):
+     * SRT extension since v1.0.1:
      *   ACKD_RCVRATE
-     * SRT extension version 1.0.4:
-     *   ACKD_XMRATE
+     * SRT extension since v1.0.2:
```
Comment (maxsharabayko) [inline srtcore/core.cpp:8010]:
```suggestion
     * SRT extension in v1.0.2 only:
```

File: srtcore/core.cpp [hunk 4]
```diff
@@ -8025,8 +8029,8 @@ void CUDT::processCtrlAck(const CPacket &ctrlpkt, const steady_clock::time_point
         m_iBandwidth        = avg_iir<8>(m_iBandwidth, bandwidth);
         m_iDeliveryRate     = avg_iir<8>(m_iDeliveryRate, pktps);
         m_iByteDeliveryRate = avg_iir<8>(m_iByteDeliveryRate, bytesps);
-        // XXX not sure if ACKD_XMRATE is of any use. This is simply
-        // calculated as ACKD_BANDWIDTH * m_iMaxSRTPayloadSize.
+        // TODO: Not sure if ACKD_XMRATE_VER102_ONLY is of any use. This is
+        // simply calculated as ACKD_BANDWIDTH * m_iMaxSRTPayloadSize.
```
Comment (maxsharabayko) [inline srtcore/core.cpp:8033]: Useless comment.
```suggestion
```

**Pull request merged**